modules/ui/notice.js

```diff
@@ -32,9 +30,20 @@ export function uiNotice(context) {
 
 
         function disableTooHigh() {
-            var canEdit = context.map().zoom() >= context.minEditableZoom()
-                && !context.map().tooManyNodes();
+            var noLayers = _isEmpty(Hoot.layers.loadedLayers);
+            var tooHigh = context.map().zoom() < context.minEditableZoom();
+            var tooManyNodes = context.map().tooManyNodes();
+            var canEdit = !tooHigh && !tooManyNodes && !noLayers;
             div.style('display', canEdit ? 'none' : 'block');
+            button.on('click', function() {
+                if (noLayers) {
+                    context.map().addDataset();
+                } else {
+                    context.map().zoomEase(16); //upstream min editable zoom
```

Comment:
Just curious, i see this moves away from zooming to `context.minEditableZoom()` and just classic zoom 16.
I think that just ends up changing us back to using 16 as opposed to [9](https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/modules/core/context.js#L178)?

Reply:
In order to display sparse data a lower zooms we changed the min editable zoom for hoot-ui to be 9.  I think rather than zoom to 9 (which may still have too many nodes) I chose to zoom to the O.G. z16